libs/data-service-generator/package.json

```diff
@@ -5,12 +5,13 @@
   "main": "./src/index.js",
   "dependencies": {
     "@amplication/code-gen-utils": "^0.0.3",
-    "@amplication/plugin-db-postgres": "1.1.7",
+    "@amplication/plugin-auth": "^1.0.2",
     "@amplication/plugin-auth-basic": "^1.2.2",
     "@amplication/plugin-auth-jwt": "^1.1.2",
     "@amplication/plugin-broker-kafka": "^0.2.2",
     "@amplication/plugin-db-mongo": "1.1.11",
     "@amplication/plugin-db-mysql": "^0.1.3",
+    "@amplication/plugin-db-postgres": "1.1.7",
```

Comment:
Check the latest version of the plugin

Reply:
But I think we decided not to work with webpack and the two last version of this plugin (before 1.1.7) are in webpack